glyphs_plist/src/font.rs

```diff
@@ -1121,11 +1142,25 @@ impl NodeType {
 
 impl ToPlist for Node {
     fn to_plist(self) -> Plist {
-        Plist::Array(vec![
-            self.pt.x.into(),
-            self.pt.y.into(),
-            self.node_type.glyphs_str().to_string().into(),
-        ])
+        let Node {
+            pt,
+            node_type,
+            attr,
+        } = self;
+
+        match attr {
+            None => Plist::Array(vec![
+                pt.x.into(),
+                pt.y.into(),
+                node_type.glyphs_str().to_string().into(),
+            ]),
+            Some(attr) => Plist::Array(vec![
+                pt.x.into(),
+                pt.y.into(),
+                node_type.glyphs_str().to_string().into(),
+                attr.to_plist(),
+            ]),
+        }
```

Comment:
Saves the duplication perhaps? (Handwritten, not checked to compile or be formatted correctly)

```suggestion
        let mut array = vec![
            pt.x.into(),
            pt.y.into(),
            node_type.glyphs_str().to_string().into(),
        ];
        if let Some(attr) = attr {
            array.push(attr.to_plist());
        }
        Plist::Array(array)
```